packages/cli/README.md

```diff
@@ -65,7 +65,7 @@ const sendTokensMsg: types.MsgSend = {
 };
 
 const signBytes = makeSignBytes([sendTokensMsg], defaultFee, defaultNetworkId, memo, account_number, sequence);
-const signature = encodeSecp256k1Signature(pen.pubkey, await pen.createSignature(signBytes));
+const signature = await pen.sign(signBytes);
```

Comment:
One method and so much cleaner. Nice

Reply:
The simplification in the usage with the RestServer made me confident that this is the better interface in general and we're not improving one case while worsen another.